lib/src/store.dart

```diff
@@ -85,8 +85,10 @@ class Store extends Disposable {
       throw new StateError('Store has been disposed');
     }
 
-    return _stream.listen(onData,
```

Comment:
I think we need to keep this as is.  I understand the rationale for proposing the change, but I think since everyone **should** be sensitive to the need to properly cancel stream subscriptions so the keyword `listen()` should already fire off their spidey sense that this is something they need to `cancel()` on their own.

i would personally find it more confusing to call `listen` on something to get a `StreamSubscription` and then NOT be supposed to `cancel` it